### ISSUE TITLE
installing all required to support fitting-type machine-pack - should be a users' choice

### ISSUE THREAD

**Issue by osher** [apigee-127/bagpipes#16]
Have a look here:
http://npm.anvaka.com/#/view/2d/bagpipes

I bet that most users won't use machine-pack. 
Using machine-pack should be a user's choice.

**Comment by theganyo** [comment]
Yes, I think that's a good point. That's actually why I marked this package "SUBJECT TO CHANGE" - because I wasn't yet comfortable with what would and would not be "standard" fittings.

**Comment by osher** [comment]
clear. I'm right behind you.
